lab_3_keywords_textrank/main.py

```diff
@@ -278,7 +330,15 @@ def calculate_inout_score(self, vertex: int) -> int:
                 number of incidental vertices
         If vertex is not present in the graph, -1 is returned
         """
-        pass
+        if vertex not in self._matrix:
+            return -1
+        count = []
+        for rows in self._matrix:
+            if rows[0] == vertex:
```

Comment:
do all your rows start with a vertex to which they relate?

if so, it was very unclear...